### DISCUSSION TITLE
Feature Req: Save buffer-local-variables

### DISCUSSION THREAD

**Discussion by mtwomey** [alphapapa/burly.el#68] [Ideas]
I have some minor modes that are influenced by buffer-local-variables. It would be very helpful if it were possible to restore these as well when opening a bookmark. Or is there already a way to do this?

**Comment by alphapapa** [comment]
This would currently be handled by the bookmarking functions, but since those apply to major modes, they would have to specifically support such minor mode variables.  Can you give me a few examples?  We might need a more general way to customize the bookmark records.  Alternatively, you could probably define advice for the relevant bookmark-make-record-function and bookmark-jump function in those buffers which would save and restore the variables.
